src/ledger/NetworkConfig.cpp

```diff
@@ -114,6 +134,18 @@ initialContractLedgerAccessSettingsEntry()
     e.bucketListWriteFeeGrowthFactor =
         InitialSorobanNetworkConfig::BUCKET_LIST_WRITE_FEE_GROWTH_FACTOR;
 
+    if (cfg.TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE)
+    {
+        e.ledgerMaxReadLedgerEntries *= 10;
+        e.ledgerMaxReadBytes *= 10;
+        e.ledgerMaxWriteLedgerEntries *= 10;
+        e.ledgerMaxWriteBytes *= 10;
+        e.txMaxReadLedgerEntries *= 10;
+        e.txMaxReadBytes *= 10;
+        e.txMaxWriteLedgerEntries *= 10;
+        e.txMaxWriteBytes *= 10;
+    }
```

Comment:
I think instead of multiplying individual setting by a scaling factor, we should define all the test configs in a new structure `TestInitialSorobanNetworkConfig`. This way is more clear and easy to maintain.